src/NzbDrone.Core/Qualities/Quality.cs

```diff
@@ -71,6 +71,8 @@ public override bool Equals(object obj)
         //public static Quality Bluray480p  { get { return new Quality(13, "Bluray-480p"); } }
         //public static Quality WEBRip720p  { get { return new Quality(14, "WEBRip-720p"); } }
         //public static Quality WEBRip1080p { get { return new Quality(15, "WEBRip-1080p"); } }
+        public static Quality REMUX1080p => new Quality(11, "REMUX-1080p");
```

Comment:
`11` is already a place holder for `public static Quality HDTV480p    { get { return new Quality(11, "HDTV-480p"); } }`